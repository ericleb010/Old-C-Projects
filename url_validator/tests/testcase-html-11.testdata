-d 2 http://www2.warnerbros.com/spacejam/movie/cmp/bball/bballframes.html
