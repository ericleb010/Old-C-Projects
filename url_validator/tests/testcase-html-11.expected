
Checking http://www2.warnerbros.com/spacejam/movie/cmp/bball/bballframes.html ... OK

>>> Traversing into http://www2.warnerbros.com/spacejam/movie/cmp/bball/bballframes.html:
|	Line 42: checking http://www2.warnerbros.com/spacejam/movie/cmp/bball/bballnoframes.html ... OK

	>>> Traversing into http://www2.warnerbros.com/spacejam/movie/cmp/bball/bballnoframes.html:
	|	Line 65: checking http://www2.warnerbros.com/spacejam/movie/cmp/bball/../../bin/bball.map ... OK
	<<< Done traversing!

<<< Done traversing!

